Hunting Queries/SecurityAlert/AlertsForIP.txt

```diff
@@ -0,0 +1,24 @@
+// Name: Alerts related to IP
+// Description: Any Alerts that fired related to a given IpAddress during the range of +6h and -3d
+//
+// Data Source: SecurityAlert
+//
+// Tactics: #Persistence, #Discovery, #LateralMovement, #Collection
+//
+// Id: 0b520385-6a16-4e6f-ba89-c320d857695f
+//
+let GetAllAlertsWithIp = (suspiciousEventTime:datetime, v_ipAddress:string){
+//-3d and +6h as some alerts fire after accumulation of events
+let v_StartTime = suspiciousEventTime-3d;
+let v_EndTime = suspiciousEventTime+6h;
+SecurityAlert
+| where TimeGenerated between (v_StartTime .. v_EndTime)
+// expand JSON properties
+| extend Extprop = parsejson(ExtendedProperties)
+| extend Computer = toupper(tostring(Extprop["Compromised Host"]))
```

Comment:
note that not all alerts in this table will have Compromised Host property or User Name

Reply:
Noted.